services/storage/object/controls.md

```diff
@@ -1,12 +1,14 @@
 # CCC.OS: Object Storage
 
-| Control Id | Service Taxonomy Id | Control                                                |
-| ---------- | ------------------- | ------------------------------------------------------ |
-| CCC.OS.C1  | CCC-020115          | Prevent unencrypted requests to object storage bucket  |
-| CCC.OS.C2  | CCC-020114          | Ensure data encryption at rest                         |
-| CCC.OS.C3  | CCC-020116          | Implement multi-factor authentication (MFA) for access |
-| CCC.OS.C4  | CCC-020112          | Maintain immutable backups of data                     |
-| CCC.OS.C5  | CCC-020118          | Log all access and changes to object storage bucket    |
+| Control Id | Service Taxonomy Id | Control                                                                        |
+| ---------- | ------------------- | ------------------------------------------------------------------------------ |
+| CCC.OS.C1  | CCC-020115          | Prevent unencrypted requests to object storage bucket                          |
+| CCC.OS.C2  | CCC-020114          | Ensure data encryption at rest                                                 |
+| CCC.OS.C3  | CCC-020116          | Implement multi-factor authentication (MFA) for access                         |
+| CCC.OS.C4  | CCC-020112          | Maintain immutable backups of data                                             |
+| CCC.OS.C5  | CCC-020118          | Log all access and changes to object storage bucket                            |
+| CCC.OS.C6  | CCC-020118          | Prevent access to object storage from trusted cloud tenants and cloud services |
```

Comment:
Should this not be "...from **un**trusted cloud tenants..."?

Reply:
This is to cater to protection from Data Exfiltration and Infiltration via Trusted Tenants and Services.